packages/sdk/src/waku/waku.ts

```diff
@@ -106,7 +107,16 @@ export class WakuNode implements IWaku {
     this.health = getHealthManager();
 
     if (protocolsEnabled.store) {
-      const store = wakuStore(this.connectionManager);
+      let peerIdStr: PeerIdStr | undefined;
+      if (options.store?.peer) {
+        this.connectionManager
+          .dialPeer(options.store.peer, [StoreCodec])
```

Comment:
let's not introduce this function and use https://github.com/waku-org/js-waku/blob/88e33a90fd2a4de93d4ce0cb99dbd77ff454ef34/packages/sdk/src/waku/waku.ts#L143

which does the same as I see